README.md

```diff
@@ -37,6 +42,9 @@ Currently the following Intel I/O Controller Hubs are supported and tested:
 | ---------------------- | -------------- | -------------------- |
 | Sunrise Point-LP (PCH) | `pci8086,9d23` | Thinkpad T480s, L380 |
 | Cannon Lake-H (PCH)    | `pci8086,a323` | Thinkpad P52         |
+| Sunrise Point-H (PCH)  | `pci8086,a123` | HP Pavilion 15       |
+| Cannon Lake-LP (PCH)   | `pci8086,9da3` | HP G. Pavilion 15    |
+
```

Comment:
Maybe order them in alphabetical order, or atleast put Cannon Lake-LP right underneath Cannon Lake-H.
Also, my X1 Extreme is 8086,a323 if we want to add that. (Can be done at another time if you want to wait for the synaptics stuff to be finished first)

Reply:
Oh, thought you were leo-labs. Ignore my comment about my X1 lol